lang/semgrep-grammars/src/semgrep-go/grammar.js

```diff
@@ -10,18 +10,28 @@ module.exports = grammar(base_grammar, {
   name: 'go',
 
   conflicts: ($, previous) => previous.concat([
+  [$.expression_statement, $.semgrep_ellipsis]
   ]),
 
   /*
      Support for semgrep ellipsis ('...') and metavariables ('$FOO'),
      if they're not already part of the base grammar.
   */
   rules: {
-    semgrep_ellipsis: $ => "...",
-
+   
     semgrep_ellipsis_metavar : $ => /\$\.\.\.[a-zA-Z_][a-zA-Z_0-9]*/,
+
+    semgrep_ellipsis: $ => "...",
     semgrep_deep_ellipsis: $ => seq("<...", $._expression, "...>"),
 
+    // 
+    _statement: ($, previous) => choice(
+      previous,
+      prec(1,$.semgrep_ellipsis_metavar),
+      prec(1,$.semgrep_deep_ellipsis),
+      prec(1,$.semgrep_ellipsis)
+    ),
```

Comment:
I am not sure if there's a better way to go about doing this via tree-sitter, but a conflict that is awkward to resolve is 
```go
if (true) {
  ...
};
```
Won't be parsed as `(if (...) (block (semgrep_elipsis)))` but instead `(if (...) (block (expression statement (semgrep_elipsis))))`; this is what this is trying to resolve